tests/_utils/run_dm_worker

```diff
@@ -25,7 +25,7 @@ PWD=$(pwd)
 echo "[$(date)] <<<<<< START DM-WORKER on port $port, config: $conf >>>>>>"
 cd $workdir
 $binary -test.coverprofile="$TEST_DIR/cov.$TEST_NAME.worker.$port.$(date +"%s").out" DEVEL \
-    --worker-addr=:$port \
+    --worker-addr=0.0.0.0:$port \
```

Comment:
do we need to check the host? `0.0.0.0:$port` will become the `advertise-addr`, but it can not be used by dm-master

Reply:
👍 addressed in https://github.com/pingcap/dm/pull/450/commits/2024c7c100401830597c2b2f847290fe4b3e73ac.
1. add the check for `0.0.0.0` as the `advertise-addr`
2. add `advertise-addr` when running dm-worker in tests.